tests/unit/amo/components/TestHeader.js

```diff
@@ -178,4 +178,19 @@ describe(__filename, () => {
 
     expect(link).toHaveLength(0);
   });
+
+  it('displays manage my submissions when user is logged in', () => {
```

Comment:
can you write: "displays a "manage my submissions" link when user is logged in"

Reply:
Done :+1: